XMonad/Actions/GridSelect.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE ScopedTypeVariables, GeneralizedNewtypeDeriving, TypeSynonymInstances, FlexibleInstances, OverlappingInstances #-}
```

Comment:
Why did the list of extensions change?  It's not obvious to me why the new functions necessitate these extensions.

Reply:
I think that is leftover from my first attempt to implement this feature, and I forgot to remove it. I will make a new pull request fixing that.